components/LanguageToggle.js

```diff
@@ -0,0 +1,149 @@
+class LanguageToggle {
+  constructor(app) {
+    this.app = app;
+    this.langButton = document.querySelector(".lang-en");
+    this.weatherCondition = document.querySelector(".weather__condition");
+    this.weatherFeelsLike = document.querySelector(".weather__feels-like");
+    this.weatherWind = document.querySelector(".weather__wind");
+    this.weatherHumidity = document.querySelector(".weather__humidity");
+    this.currentLanguage = "en";
+    this.latit = document.querySelector(".latitude-text");
+    this.langit = document.querySelector(".longitude-text");
+    this.firstDaylang = document.querySelector(".first-day");
+    this.secondDaylang = document.querySelector(".second-day");
+    this.lastDaylang = document.querySelector(".last-day");
+    this.dateElement = document.querySelector(".date");
+
+    this.translations = {
+      en: {
+        overcast: "OVERCAST",
+        feelsLike: "FEELS LIKE",
+        wind: "WIND",
+        humidity: "HUMIDITY",
+        latitude: "latitude",
+        longitude: "longitude",
+        days: {
+          0: "Sunday",
+          1: "Monday",
+          2: "Tuesday",
+          3: "Wednesday",
+          4: "Thursday",
+          5: "Friday",
+          6: "Saturday",
+        },
+        months: {
+          0: "January",
+          1: "February",
+          2: "March",
+          3: "April",
+          4: "May",
+          5: "June",
+          6: "July",
+          7: "August",
+          8: "September",
+          9: "October",
+          10: "November",
+          11: "December",
+        },
+      },
+      ru: {
+        overcast: "ПАСМУРНО",
+        feelsLike: "ОЩУЩАЕТСЯ КАК",
+        wind: "ВЕТЕР",
+        humidity: "ВЛАЖНОСТЬ",
+        latitude: "широта",
+        longitude: "долгота",
+        days: {
+          0: "Воскресенье",
+          1: "Понедельник",
+          2: "Вторник",
+          3: "Среда",
+          4: "Четверг",
+          5: "Пятница",
+          6: "Суббота",
+        },
+        months: {
+          0: "Январь",
+          1: "Февраль",
+          2: "Март",
+          3: "Апрель",
+          4: "Май",
+          5: "Июнь",
+          6: "Июль",
+          7: "Август",
+          8: "Сентябрь",
+          9: "Октябрь",
+          10: "Ноябрь",
+          11: "Декабрь",
+        },
+      },
+    };
```

Comment:
Move to shared/translations-en and ru files

Reply:
https://github.com/VasiliiDubik/Vasiliidubik-weatherForecast/pull/1/commits/22153b0d01a31358587e8d43b7fca83e285d3c0c

Reply:
https://github.com/VasiliiDubik/Vasiliidubik-weatherForecast/pull/1/commits/22153b0d01a31358587e8d43b7fca83e285d3c0c